index.js

```diff
@@ -27,6 +27,7 @@ module.exports = {
     const IS_PREVIEW = process.env.CONTEXT == 'deploy-preview'
 
     /* Set the user input settings */
+    const sentryUrl = process.env.SENTRY_URL || inputs.sentryUrl || "https://sentry.io/";
```

Comment:
@baffo by default in the sentry-cli we set the `SENTRY_URL` to "https://sentry.io/" if it is not set. https://github.com/getsentry/sentry-cli/blob/38f7f254c03f4e3681463881a2eb54266d08ea5e/src/config.rs#L545

I think it's best to have one source of truth for the defaults and use sentry-cli.

Reply:
I agree with the suggestion @NisanthanNanthakumar, I removed the literal declaration of the url as it's unnecessary